sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveContext.scala

```diff
@@ -95,12 +95,12 @@ class HiveContext(sc: SparkContext) extends SQLContext(sc) {
    */
   def refreshTable(tableName: String): Unit = {
     // TODO: Database support...
-    catalog.refreshTable("default", tableName)
+    catalog.refreshTable(Seq("default", tableName))
   }
 
   protected[hive] def invalidateTable(tableName: String): Unit = {
     // TODO: Database support...
-    catalog.invalidateTable("default", tableName)
+    catalog.invalidateTable(Seq("default", tableName))
```

Comment:
yes, this just inherited from the old version. Now ```invalidateTable``` do not support database as you suggest, so here is a todo in line 102, how about make another PR to fix this? or do it in this PR